lib/bunq/user.rb

```diff
@@ -47,8 +48,12 @@ def draft_share_invite_banks
       Bunq::DraftShareInviteBanks.new(@resource)
     end
 
-    def certificate_pinned
-      Bunq::CertificatePinned.new(@resource)
+    def certificate_pinned(id)
```

Comment:
Beware that this breaks backward compatibility. In order to fix that you could return a `Bunq::CertificatesPinned` here whenever this method is called without arguments..

Reply:
True, I add this to the CHANGELOG.md